src/js/templates/maps/viewfinder/viewfinder-prediction.html

```diff
@@ -0,0 +1,6 @@
+<div class="<%= classNames.content %> <% if(isFocused) { %>viewfinder-prediction__focused<% } %>">
+  <i class="icon-map-marker"></i>
+  <div class="viewfinder-prediction__main" title="<%= description %>">
```

Comment:
This provides a mechanism for the user to see the entire prediction description, since it will overflow with ellipsis if the string is too long
![ellipses](https://github.com/NCEAS/metacatui/assets/4664309/d2526713-d71b-4e43-96d4-f6712431f264)